packages/blaze/package.js

```diff
@@ -24,14 +26,14 @@ Package.onUse(function (api) {
   api.export([
     'Blaze',
     'UI',
```

Comment:
UI can be removed as deprecated

Reply:
As the previous comment

Reply:
In the meantime, I did the push with the changes without no-param-reassign. If you take a look I will also fix any other points.